exploration/open-close-expressions.md

```diff
@@ -220,3 +220,68 @@ Rendered as React this could become:
 _What other solutions are available?_
 _How do they compare against the requirements?_
 _What other properties they have?_
+
+### Do Nothing
+
+We could not provide any special support for spans and markup elements,
+and instead delegate the problem completely to tools and downstream processing layers.
```

Comment:
"could not" is hard to parse?

```suggestion
Do not provide any special support for spannables or markup.
This would delegate the problem to tools and downstream processing layers.
```